docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
```

Comment:
why cbc?

Reply:
No idea how that happened. Will remove.

Reply:
That line was added by you :-) https://github.com/JuliaPlots/GraphMakie.jl/pull/83/commits/b8ba9ada5a692a0a45f72bcad7d4e4f69d9d0f71

It shows up here because I merged master into this branch.

Reply:
https://github.com/JuliaPlots/GraphMakie.jl/pull/78
ah yes, i remeber now, this was a hotfix to adress Makie@0.17 compat